x-pack/solutions/security/plugins/session_view/public/hooks/use_eui_theme.ts

```diff
@@ -33,10 +33,10 @@ export const useEuiTheme = (...props: EuiThemeProps): EuiThemeReturn => {
 
     const extraEuiVars: ExtraEuiVars = {
       // eslint-disable-next-line @typescript-eslint/naming-convention
-      euiColorVis6_asText: shade(themeVars.euiColorVis6, 0.335),
-      buttonsBackgroundNormalDefaultPrimary: '#006DE4',
+      euiColorVis6_asText: shade(themeVars.euiColorVis6, 0.335), // TODO: check in Borealis. Replce with proper color token and remove shade
```

Comment:
`euiColorVis6` becomes pink in Borealis and there is no matching color to the Amsterdam color in the new palette. requires discussion with designers